charts/tidb-drainer/values.yaml

```diff
@@ -48,6 +48,22 @@ tlsCluster:
   certAllowedCN: []
   #  - TiDB
 
+# The tls config between drainer and the downstream database server (MySQL/TiDB)
+tlsSyncer:
+  tlsClientSecretName:
+
+  # certAllowedCN is the Common Name that allowed
```

Comment:
What's the difference between this one and the one in L64, please make it clear.